pkgs/development/compilers/sbcl/bootstrap.nix

```diff
@@ -23,7 +23,6 @@ let
       sha256 = "0sp5445rbvms6qvzhld0kwwvydw51vq5iaf4kdqsf2d9jvaz3yx5";
     };
     armv6l-linux = armv7l-linux;
-    x86_64-solaris = x86_64-linux;
```

Comment:
I think Illumos has some basic Linux ABI compatibility. So I think `solaris` bootstrap can be either switched to a native Solaris-targeted binary or left as-is.